### ISSUE TITLE
Component Accessibility Status: Final Dev Deliverable

### ISSUE THREAD

**Issue by laurenmrice** [carbon-design-system/carbon-website#3519]
**Developer** | Will work on showing this final design on the website while working closely with a designer (Thy) 

- [ ] Task 1 (dev defines the work here)
- [ ] Task 2 (dev defines the work here)
- [ ] Task 3 (dev defines the work here)

**Comment by tay1orjones** [comment]
As part of this, dev should explore how to surface reporting of what types of tests a component has via playwright's `grep` tags. We may need to add new tags, `@avt`, `@avt-complex-states`, `@avt-keyboard-nav`, `@vrt`, etc.

By gathering these tests per-component we should be able to produce a JSON file or something that contains data giving us the ability to build a table something like

| component | `@avt` | `@avt-complex-states` |
|--------|--------|--------|
| Dropdown | ✅ | 🚫 |
| MultiSelect | ✅ | ✅ |
| Button | 🚫 | ✅ |

**Comment by tay1orjones** [comment]
Another concern brought up was that as we look at this, we should take into account that these tests are only for `@carbon/react` and the design needs to scale to include other framework implementations, such as `@carbon/web-components`.

**Comment by tay1orjones** [comment]
Looking forward, this issue will contain the work that comes out of the design explorations:

* https://github.com/carbon-design-system/carbon/issues/14484
* https://github.com/carbon-design-system/carbon-website/issues/3518

**Comment by sstrubberg** [comment]
closing in favor of the issues created yesterday in refinement.
